<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quickview</class>
 <widget class="QWidget" name="quickview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="mDayBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Day</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mWeekBtn">
       <property name="text">
        <string>Week</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mMothBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Month</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mAgendaBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Agenda</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mPreviousBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mTodayBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Today</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mNextBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="calender">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
